### PR TITLE
Get complementary feeding form data from the new form table

### PR DESCRIPTION
**Pull request by emord**
Makes it so that we no longer use complementary feeding data from the monthly UCR. Once this has been deployed can [remove those calculations from the UCR](https://github.com/dimagi/commcare-hq/commit/02d7c3b41e41477cfdbcc1e3554047c4a5a66428)

Lots of +/- here, but it's because I needed to rewrite the test CSV files with [new columns](https://github.com/dimagi/commcare-hq/commit/fdc94e6740d956dd74e81c4ca10d3a2d48ed8483) and fixed a [bit of a bug](https://github.com/dimagi/commcare-hq/commit/43867f70560725151255ab29546a29e786eef79d)

:blowfish: / :blowfish:

### CODE REVIEW

**Comment by emord** [comment]
(looking into the test failure)

**Review by snopoke — COMMENTED** [review]
Didn't have context on all the fields but generally looks good.

**Review by dannyroberts — APPROVED** [review]

File: custom/icds_reports/migrations/0035_auto_20180317_0645.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('icds_reports', '0035_add_pnc_aggregate_models'),
+    ]
+
+    operations = [
+        migrator.get_migration('update_tables17.sql'),
```
Comment (lwyszomi) [inline custom/icds_reports/migrations/0035_auto_20180317_0645.py]: Hi @emord I think you should add in this place also ```migrator.get_migration('create_functions.sql')``` without this, we will have the old ```insert_into_child_health_monthly``` definition on the production.

Comment (emord) [reply custom/icds_reports/migrations/0035_auto_20180317_0645.py]: @lwyszomi we run `create_functions` every time we aggregate https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/tasks.py#L83-L96

Likely something that should be changed, but for now it's not an issue

Comment (lwyszomi) [reply custom/icds_reports/migrations/0035_auto_20180317_0645.py]: @emord didn't notice. I thought that the task runs only aggregation functions.

**Pull request merged**